assets/js/base/components/reviews/review-sort-select/index.tsx

```diff
@@ -2,15 +2,25 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import PropTypes from 'prop-types';
 import SortSelect from '@woocommerce/base-components/sort-select';
+import { ChangeEventHandler } from 'react';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
 
-const ReviewSortSelect = ( { onChange, readOnly, value } ) => {
+interface ReviewSortSelectProps {
+	onChange: ChangeEventHandler;
```

Comment:
It seems that onChange can be optional. You can achieve it in TS in this way:
```suggestion
	onChange?: ChangeEventHandler;
```

Reply:
> It seems that onChange can be optional. You can achieve it in TS in this way:

I'm not sure, the onChange function is a required prop in SortSelect: assets/js/base/components/sort-select/index.tsx

I tried changing both of them to optional. It was creating more issues, so I rolled back to required for now. Is there any other way to make them optional?